sentry-android-core/src/test/java/io/sentry/android/core/ANRWatchDogTest.kt

```diff
@@ -76,4 +81,83 @@ class ANRWatchDogTest {
             es.shutdown()
         }
     }
+
+    @Test
+    fun `when ANR is detected and ActivityManager has ANR process, callback not invoked`() {
```

Comment:
Is this the name instead?

```suggestion
    fun `when ANR is detected and ActivityManager has ANR process, callback is invoked`() {
```

Reply:
damn good catch, I need to pay more attention to my copypasta hehe